### PR TITLE
Write add transceiver section. Fixes #299

### LINKED ISSUE #299: Add section for addTransceiver
In the current draft we have 

4.1.3.  addTransceiver

   [TODO]

We need this text before WGLC, so it's being added to the punchlist.

### PR DESCRIPTION
**Pull request by ekr**
@pthatcherg

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1161,7 +1161,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="addTransceiver" anchor="sec.addTransceiver">
-                  <t>[TODO]</t>
+                  <t>
+                    The addTransceiver method adds a transceiver to the PeerConnection.
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: perhas: 'an RTPTransceiver'

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -1161,7 +1161,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                    If a MediaStreamTrack argument is provided, then the transceiver will
+                    be configured with that media type and attached to that track. Otherwise,
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Can a transceiver be attached to a track? I think it's the other way around.

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -1161,7 +1161,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                    the application MUST explicitly specify the type; this mode is useful
+                    for creating recvonly transceivers as well as for creating transceivers
+                    which will be attached to a track at some later point.
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: re: attached; see above

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -1161,7 +1161,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                    At the time of creation, the application can also specify a transceiver
+                    direction attribute, a set of MediaStreams which the transceiver is
+                    associated with (allowing LS group assignments),
+                    and a set of encodings for the media (used for simulcast [TODO section]).
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Do you want to expand on how the direction and MediaStreams are used? I think spelling this out would be useful.

**Comment by ekr** [comment]
@juberti PTAL

File: draft-ietf-rtcweb-jsep.xml [hunk 5]
```diff
@@ -1161,7 +1161,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="addTransceiver" anchor="sec.addTransceiver">
-                  <t>[TODO]</t>
+                  <t>
+                    The addTransceiver method adds a new
+                    RTPTransceiver to the PeerConnection.  If a MediaStreamTrack argument
+                    is provided, then the transceiver will be configured with that media
+                    type and the track is attached to the transceiver. Otherwise, the
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: is -> will be

File: draft-ietf-rtcweb-jsep.xml [hunk 6]
```diff
@@ -1161,7 +1161,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                    application MUST explicitly specify the type; this mode is useful for
+                    creating recvonly transceivers as well as for creating transceivers to
+                    which a track can be attached at some later point.  At the time of
+                    creation, the application can also specify a transceiver direction
+                    attribute, a set of MediaStreams which the transceiver is associated
+                    with (allowing LS group assignments), and a set of encodings for the
+                    media (used for simulcast [TODO section]).
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: fixup section, which now exists

File: draft-ietf-rtcweb-jsep.xml [hunk 7]
```diff
@@ -1161,7 +1161,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
-                  <t>[TODO]</t>
+                  <t>
+                    The addTransceiver method adds a new
+                    RTPTransceiver to the PeerConnection.  If a MediaStreamTrack argument
+                    is provided, then the transceiver will be configured with that media
+                    type and the track is attached to the transceiver. Otherwise, the
+                    application MUST explicitly specify the type; this mode is useful for
+                    creating recvonly transceivers as well as for creating transceivers to
+                    which a track can be attached at some later point.  At the time of
+                    creation, the application can also specify a transceiver direction
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Suggest paragraph break for this final sentence

**Comment by juberti** [comment]
LG with nits

File: draft-ietf-rtcweb-jsep.xml [hunk 8]
```diff
@@ -1161,7 +1161,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                    RTPTransceiver to the PeerConnection.  If a MediaStreamTrack argument
+                    is provided, then the transceiver will be configured with that media
+                    type and the track will be attached to the transceiver. Otherwise, the
+                    application MUST explicitly specify the type; this mode is useful for
+                    creating recvonly transceivers as well as for creating transceivers to
+                    which a track can be attached at some later point.  At the time of
+                    creation, the application can also specify a transceiver direction
+                    attribute, a set of MediaStreams which the transceiver is associated
+                    with (allowing LS group assignments), and a set of encodings for the
+                    media (used for simulcast as desccribed in <xref target="sec.simulcast"/>).
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: typo: desccribed

**Comment by juberti** [comment]
para break and typo comment were missed

**Pull request merged**